cfme/common/__init__.py

```diff
@@ -321,19 +321,28 @@ def _tags_action(self, view, cancel, reset):
 class Taggable(TaggableCommonBase):
     """
     This class can be inherited by any class that honors tagging.
-    Class should have following
+
+    Classes inheriting this class should have following defined:
 
     * 'Details' navigation
     * 'Details' view should have entities.smart_management SummaryTable widget
-    * 'EditTags' navigation
-    * 'EditTags' view should have nested 'form' view with 'tags' table widget
+    * 'EditTagsFromDetails' navigation
+    * 'EditTagsFromDetails' At a minimum, views should include EditTagsFromDetails class. View
```

Comment:
Q: Can you explain to me this part? I checked a few classes that inherit `Taggable` and I don't see this navigation defined.

Reply:
Absolutely. Here is an example:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/__init__.py#L498

I have observed a couple places where we have the class name wrong. The class has a preq of navigating to details but the class name is `EditTags`. Here is an example of that:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/containers/replicator.py#L107

`EditTags` should have the preq of navigating to `All`. I should add that into to the doc block.

Reply:
Thanks for the explanation!